### PR TITLE
added thousands separator to MoneyUnit for readability

### PR DESCRIPTION
**Pull request by soraphis**

### CODE REVIEW

File: NBitcoin/Money.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿using System.Globalization;
+using System.Globalization;
```
Comment (lontivero) [inline NBitcoin/Money.cs]: This is not necessary.

Comment (nopara73) [reply NBitcoin/Money.cs]: What is this btw? I see this glitch here and there.

Comment (knocte) [reply NBitcoin/Money.cs]: it's the BOM being changed accidentally by text editors

Comment (soraphis) [reply NBitcoin/Money.cs]: I don't get why this project i so broken in regards to line endings

**Pull request merged**